pkg/client/unstructured_client.go

```diff
@@ -142,7 +142,7 @@ func (uc *unstructuredClient) UpdateStatus(_ context.Context, obj runtime.Object
 	if err != nil {
 		return err
 	}
-	i, err := r.UpdateStatus(u)
+	i, err := r.UpdateStatus(u, metav1.UpdateOptions{})
```

Comment:
@droot we'll want to surface create options somewhere in our client at some point